src/dialogs/more-info/ha-more-info-dialog.ts

```diff
@@ -0,0 +1,298 @@
+    }
+    const entityId = this.hass.moreInfoEntityId;
+    const stateObj = this.hass.states[entityId];
+    const domain = computeDomain(entityId);
+
+    return html`
+      <ha-dialog open @closed=${this._close} .heading=${true} hideActions>
+        <app-toolbar slot="heading">
+          <mwc-icon-button
+            .label=${this.hass.localize("ui.dialogs.more_info_control.dismiss")}
+            dialogAction="cancel"
+          >
+            <ha-svg-icon .path=${mdiClose}></ha-svg-icon>
+          </mwc-icon-button>
+          <div class="main-title" main-title @click=${this._enlarge}>
+            ${stateObj ? computeStateName(stateObj) : ""}
+          </div>
+          ${this.hass.user!.is_admin
+            ? html`<mwc-icon-button
+                .label=${this.hass.localize(
+                  "ui.dialogs.more_info_control.settings"
+                )}
+                @click=${this._gotoSettings}
+              >
+                <ha-svg-icon .path=${mdiSettings}></ha-svg-icon>
+              </mwc-icon-button>`
+            : ""}
+          ${this.hass.user!.is_admin &&
+          ((EDITABLE_DOMAINS_WITH_ID.includes(domain) &&
+            stateObj?.attributes.id) ||
+            EDITABLE_DOMAINS.includes(domain))
+            ? html` <mwc-icon-button
+                .label=${this.hass.localize(
+                  "ui.dialogs.more_info_control.edit"
+                )}
+                @click=${this._gotoEdit}
+              >
+                <ha-svg-icon .path=${mdiPencil}></ha-svg-icon>
+              </mwc-icon-button>`
+            : ""}
+        </app-toolbar>
+        <div class="content">
+          ${DOMAINS_NO_INFO.includes(domain)
+            ? ""
+            : html`
+                <state-card-content
+                  .stateObj=${stateObj}
+                  .hass=${this.hass}
+                  in-dialog
+                ></state-card-content>
+              `}
+          ${this._computeShowHistoryComponent(entityId)
+            ? html`
+                <state-history-charts
+                  .hass=${this.hass}
+                  .historyData=${this._stateHistory}
+                  upToNow
+                  .isLoadingData=${!this._stateHistory}
+                ></state-history-charts>
+              `
+            : ""}
+          <more-info-content
+            .stateObj=${stateObj}
+            .hass=${this.hass}
+          ></more-info-content>
+
+          ${stateObj?.attributes.restored
+            ? html`${this.hass.localize(
+                  "ui.dialogs.more_info_control.restored.not_provided"
+                )} <br />
+                ${this.hass.localize(
+                  "ui.dialogs.more_info_control.restored.remove_intro"
+                )} <br />
+                <mwc-button class="warning" @click=${this._removeEntity}
+                  >${this.hass.localize(
+                    "ui.dialogs.more_info_control.restored.remove_action"
+                  )}</mwc-button
+                >`
```

Comment:
I find it a little difficult to read this block. Can we try to put a newline after we start `html` blocks?